Changes

```diff
@@ -165,6 +165,10 @@ Working version
 - GPR#1585: optimize output of "ocamllex -ml"
   (Alain Frisch, review by Frédéric Bour and Gabriel Scherer)
 
+- GPR#1537: boot/ocamldep is no longer included in the distribution;
```

Comment:
Could you add "in the **source** distribution" to make it clear that binary distributions are not affected by this change?

Reply:
Fixed, thanks.